packages/react-components/icons/Times.tsx

```diff
@@ -0,0 +1,38 @@
+import colors from '@celo/react-components/styles/colors.v2'
+import * as React from 'react'
+import Svg, { Path } from 'svgs'
+
+export interface Props {
+  height?: number
+  color?: string
+  strokeWidth?: number
+}
+
+function Times({ color, height, strokeWidth }: Props) {
```

Comment:
nit: Would "Close" be a more descriptive name for this icon?

Reply:
Usually I see this icon under "times" name - https://fontawesome.com/icons?d=gallery&q=times